index.html

```diff
@@ -9298,6 +9298,279 @@
+               <code><a>bufferedAmount</a></code> attribute by the length of the
+               <code>ArrayBufferView</code> in bytes.</li>
+               <li>Queue <var>data</var> for transmission on <var>stream</var>'s
+               underlying data transport. If queuing <var>data</var> is not
+               possible because not enough buffer space is available, <a>throw</a>
+               an <code>OperationError</code>.</p>
+               <div class="note">The actual transmission of data occurs in
+               parallel. If sending data leads to a QUIC-level error, the
+               application will be notified asynchronously through the
+               <code><a>RTCQuicTransport</a></code>'s <code><a>onerror</a></code>.</div></li>
+              <table class="parameters">
+                <tbody>
+                  <tr>
+                    <th>Parameter</th>
+                    <th>Type</th>
+                    <th>Nullable</th>
+                    <th>Optional</th>
+                    <th>Description</th>
+                  </tr>
+                  <tr>
+                    <td class="prmName">data</td>
+                    <td class="prmType"><code>ArrayBuffer</code></td>
+                    <td class="prmNullFalse"><span role="img" aria-label=
+                    "False">&#10008;</span></td>
+                    <td class="prmOptFalse"><span role="img" aria-label=
+                    "False">&#10008;</span></td>
+                    <td class="prmDesc"></td>
+                  </tr>
+                </tbody>
+              </table>
+              <div>
+                <em>Return type:</em> <code>void</code>
+              </div>
+            </dd>
+            <dt><dfn><code>read</code></dfn></dt>
+            <dd>
+              <p>Reads from <code><a>RTCQuicStream</a></code>. When the <code>read()</code> method is called,
+              the user agent <em class="rfc2119" title="MUST">MUST</em> run the following
+              steps:</p>
+              <p>1. Let <var>stream</var> be the <code><a>RTCQuicStream</a></code> object on which <code>read()</code>
+              is invoked.</p>
+              <p>2. If <var>stream</var>'s  <code>state</code> is <code>closing</code> or <code>closed</code>, then abort
+              these steps.</p>
+              <p>3. If <var>stream</var>'s <a>[[\Readable]]</a> slot is <code>false</code>, <a>throw</a> an <code>InvalidStateError</code>.</p>
+              <div>
+                <em>No parameters.</em>
+              </div>
+              <div>
+                <em>Return type:</em> <code>ArrayBuffer</code>
+              </div>
+            </dd>
+      </section>
+      </div>
+    </section>
+    <section id="rtcquicstreamstate*">
+      <h2><dfn>RTCQuicStreamState</dfn> Enum</h2>
+      <p>The <code>RTCQuicStreamState</code> provides information
+      on the state of the QUIC stream.</p>
+      <div>
+        <pre class="idl">enum RTCQuicStreamState {
+    "idle",
+    "open",
```

Comment:
When does something go into "open"?  It seems like we would just go straight to "open".  

I could see some value in knowing that the remote side acked that you created the stream.  That doesn't match the IETF doc, but we can make our own states.  Perhaps:

"creating"  // created locally
"created"  // remote side acked that local side created, or remote side created
"closing"  // closed locally
"closed"  // remote side acked the local side closed, or remote side closed

Reply:
@pthatcherg  The quicTransport.createStream() .state may not immediately be "open" if quicTransport.state was "new" or "connecting" when createStream was called.

Reply:
Have added "opening" state, as well as text tying the `RTCQuicStreamState` to the states defined in the QUIC Transport document Section 10.2.  For example:
1. "new" maps to the "idle" state in the QUIC transport document.
2. "open" maps to the "open" state in the QUIC transport document, except for when a STREAM frame is sent but not acknowledged yet (which maps to "opening"). 
3. "closing" maps to the "half closed" states in QUIC transport.
4. "closed" maps to the "closed" state in QUIC transport.

Reply:
So 

new == haven't told other side
opening == told other side; not acked
open == both sides know 

I'm fine with that.

But I'm not so sure about "closing"